src/scripts/main.js

```diff
@@ -1,3 +1,30 @@
 'use strict';
 
-// write your code here
+const logo = document.querySelector('.logo');
+
+if (logo) {
+  const messageDiv = document.createElement('div');
+
+  messageDiv.classList.add('message');
+
+  function handleLogoClick() {
+    setTimeout(() =>{
+    const promise1 = Promise.resolve('Promise was resolved!');
+
+      promise1.then((result) => {
+        messageDiv.innerText = result;
+        document.body.appendChild(messageDiv);
+      });
+    }, 0)
+
+    setTimeout(() => {
+      const promise2 = Promise.reject(new Error('Promise was rejected!'));
+      promise2.catch((error) => {
+        messageDiv.classList.add('error-message');
+        messageDiv.innerText = error.message;
+      });
+    }, 3000);
```

Comment:
You are creating a new promise that is immediately rejected. This is not a common use case for promises. Promises are typically used for handling asynchronous operations where you don't know when the operation will complete. If you want to simulate an error condition, consider throwing an error instead.